test/acceptance/acceptance.js

```diff
@@ -3,7 +3,7 @@
 const chai = require('chai');
 const chaiAsPromised = require('chai-as-promised');
 const { expect } = require('chai');
-const AWS = require('aws-sdk');
+const Lambda = require('aws-sdk/clients/lambda');
```

Comment:
This change is probably not required but 🤷‍♂️  
Might be a good idea to add an eslint rule that asserts no imports are made from `aws-sdk` - though it will not work for the various dynamic requires